CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ access token. This enables keeping a session alive for longer than the lifetime
 of a single access token.
 - The `Session` class now exposes an `onError` method, which is a hook where
 error-handling callbacks may be registered.
+- The `Session` class now exposes an `onSessionExpiration` method, which is a hook
+where a callback may be registered to handle session expiration in the case when
+silent authentication fails.
```

Comment:
Good point, I didn't remember but `setSessionInfo` does set a callback calling the logout function. I'll change this so that it no longer does that, and so that `EVENTS.SESSION_EXPIRED` is sent even if no refresh token is issued.

Reply:
Sounds good. Does that callback also do other things, e.g. clearing cookies or localStorage entries or something? If so, maybe that's something we want to do here as well.